doc/nrf/libraries/caf/sensor_manager.rst

```diff
@@ -318,3 +318,19 @@ Sending :c:struct:`wake_up_event` to other modules results in waking up the whol
    Do not include the configuration file in other source files.
 .. |device_pm_note| replace:: Not all device power states might be supported by the sensor's device.
    Check the sensor's driver implementation before configuring :c:member:`sm_sensor_config.suspend`.
+
+Changing sensors sample period
+==============================
+
+In order to change sensors sample period user has to send :c:struct:`set_sensor_period_event` with new periods value in ms.
+To identify which sensors sampling period you want to change pass sensors description in :c:struct:`set_sensor_period_event`.
+Example code changing accelerometer sampling to 400 ms.
```

Comment:
The following code shows an example of changing accelerometer sampling to 400 ms: